benchmarks/nx-cugraph/pytest-based/README.md

```diff
@@ -0,0 +1,45 @@
+## `nx-cugraph` Benchmarks
+
+### Overview
+
+This directory contains a set of scripts designed to benchmark NetworkX with the `nx-cugraph` backend and deliver a report that summarizes the speed-up and runtime deltas over default NetworkX.
+
+Our current benchmarks provide the following datasets:
+
+| Dataset     | Nodes | Edges | Directed |
+| --------    | ------- | ------- | ------- |
+| netscience  | 1,461    | 5,484 | Yes |
+| email-Eu-core  | 1,005    | 25,571 | Yes |
+| cit-Patents  | 3,774,768    | 16,518,948 | Yes |
+| hollywood  | 1,139,905    | 57,515,616 | No |
+| soc-LiveJournal1  | 4,847,571    | 68,993,773 | Yes |
+
+
+
+### Scripts
+
+#### 1. `run-gap-benchmarks.sh`
```

Comment:
I like that we're using many of the same algos covered by the GAP benchmark, but I think it might be misleading to refer to this as "gap" since GAP also specifies the graph inputs, has a reference implementation to compare to, and possibly other things a user might expect to see.

Can we use another name for this script, perhaps just "run-main-benchmarks.sh" or something to indicate it's just the top n algos we've selected?